.github/workflows/mkdocs-deploy.yml

```diff
@@ -1,4 +1,4 @@
-name: MkDocs Deploy
+name: Deploy MkDocs Site
```

Comment:
Please keep pull requests focused - our documentation deployment isn't part of our continuous integration.

Reply:
Oh okay, will fix!

Reply:
Yeah all changes should be inside of `ci.yml` exclusively for this PR - any changes you want to make to `mkdocs-deploy.yml` should really be in a dedicated PR - though it just looks like you renamed stuff so probably not worth doing unless there was a functional difference. Probably best to just revert all changes to `mkdocs-deploy.yml` here.

Reply:
Will do! If it's necessary I could make another PR but I think it'll be fine.